src/cas/utils/conversion_utils.py

```diff
@@ -271,3 +281,23 @@ def json_serializer(obj):
         # return obj.isoformat()
         return obj.strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3] + "Z"
     raise TypeError("Type %s not serializable" % type(obj))
+
+
+def get_authors_from_doi(doi):
+    response = requests.get(f"{CROSSREF_API_URL}{doi}")
```

Comment:
Should we add some doi pre-processing here? Will this work with all possible doi formats: `https://doi.org/10.1080/14729679.2018.1507831`, `http:/dx.doi.org/10.1080/14729679.2018.1507831`, `DOI:10.1080/14729679.2018.1507831` and `10.1080/14729679.2018.1507831`

Reply:
The API is compatible with all the provided DOI formats.